adapters/vox/vox.go

```diff
@@ -0,0 +1,63 @@
+package vox
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	requestJSON, err := json.Marshal(request)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	requestData := &adapters.RequestData{
+		Method: "POST",
+		Uri:    a.endpoint,
+		Body:   requestJSON,
+	}
+
+	return []*adapters.RequestData{requestData}, nil
+}
+
+func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if responseData.StatusCode != http.StatusOK {
+		return nil, nil
+	}
+
+	var response openrtb2.BidResponse
+	err := json.Unmarshal(responseData.Body, &response)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
+	bidResponse.Currency = response.Cur
+
+	for _, seatBid := range response.SeatBid {
+		for _, bid := range seatBid.Bid {
+			b := &adapters.TypedBid{
+				Bid: &bid,
+			}
```

Comment:
Requesting to please add the BidType in the `adapters.TypedBid` response.
Here is an [excerpt from docs](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#object-model) - 
```
Prebid Server defined value identifying the media type as banner, video, audio, or native. Should be mapped from the bidding server response.
```

Also I would like to request to please define this value based on the response from bidder server. The ideal way is to set the `openrtb2.Bid.MType` field. [Here is an example on how to do this. ](https://github.com/prebid/prebid-server/blob/f275f3287f644293d102c07be4306a6b81cf2642/adapters/aidem/aidem.go#L89-L102)